sdk-commons/src/main/java/org/openmrs/maven/plugins/model/Server.java

```diff
@@ -333,18 +337,17 @@ private boolean linkProject(Project project) {
             if (Files.isSymbolicLink(linkPath)) {
                 try {
                     if (!Files.isSameFile(Paths.get(project.getPath()), linkPath)) {
-                        System.out.println("\nDeleting a link at " + link.getAbsolutePath() + " as it points to a different location.");
+                        logger.info("\nDeleting a link at {} as it points to a different location.", link.getAbsolutePath());
                         Files.delete(linkPath);
                     } else {
                         return true;
                     }
                 } catch (IOException e) {
-                    System.out.printf((CANNOT_CREATE_LINK_MSG) + "%n", link.getAbsolutePath(), e.getMessage());
+                    logger.error(CANNOT_CREATE_LINK_MSG, link.getAbsolutePath(), e.getMessage());
```

Comment:
It is usually great to pass the exception object at the end

Reply:
I made the changes